### ISSUE TITLE
Do you still have interest in this?

### ISSUE THREAD

**Issue by mame98** [antoniovazquezblanco/ArchSetup#34]
I think I will finally have some time to look back into this. I would like to test it tomorrow and based on that I would want to start working on this again...

**Comment by antoniovazquezblanco** [comment]
Yes I am. It has been long since the last time I touched this so I don't know if any other alternatives with better code have appeared. Unfortunately I do not have much time to spend on this.

Thank you for getting in touch again!

**Comment by mame98** [comment]
Ok, So I would do a basic test today and will try to work on the Logging issue ( #33  ) first

**Comment by mame98** [comment]
So, after a first test run I can say:

1) Most things work
2) There is a missing import in Interface.py (fixed this, will submit PR together with logging)
3) addwstr does not really work well (see #35 )
